docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A fresh approach for date and time inputs
+description: Check out what's new with the next-gen pickers in earlier v6 versions.
+date: 2022-12-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+---
+
+There's a lot of exciting news in MUI X v6, but there's hardly anything comparable to the revamp we're delivering for the [Date and Time Pickers](https://next.mui.com/x/react-date-pickers/getting-started/).
+
+This is the result of our ongoing efforts to improve the usability and customizability of these components.
+We kept the best aspects of the previous versions, and we're fleshing out the package with several improvements and one fundamental change: a new input type to replace the masked text fields.
+
+## The new date and time fields
+
+This new input type we're talking about is a set of components shortly named [Fields](https://next.mui.com/x/react-date-pickers/fields/).
+They are the base for the new pickers and a revolution in terms of keyboard usability.
+
+In contrast with the masked inputs that only display simple text behind a formatting mask, and were often cumbersome, especially when editing dates, the new fields are smart and specialized in editing date and time numbers.
+They're "smart" in the sense that they can recognize when a user is inputting any given piece of time data—a month, a day, a year—and respond accordingly.
+
+For example, watch what happens when you edit the month using the arrow keys:
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/intelligent-field.mp4" type="video/mp4" />
+</video>
+
+As you increase the month, the component automatically watches for valid values throughout each piece of your date input, like you're navigating in a calendar.
+
+The Fields are a perfect match to the calendar view, but they are available as stand-alone components, and you can import them from the latest v6 package.
+
+```jsx
+import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
```

Comment:
You can remove all the `Unstable` from the blog post for both the fields and the pickers.

For the field, they don't have the prefix anymore on `next` (will be released this week).

For the pickers, we are renaming it to their final name before releasing this blog post

```suggestion
import { DateField } from '@mui/x-date-pickers/DateField';
```